com/win32comext/mapi/src/mapi_headers/MAPIOID.h

```diff
@@ -1,106 +1,107 @@
-/*
- *  M A P I O I D . H
- *
- *  MAPI OID definition header file
- *
- *  Copyright (c) 2009 Microsoft Corporation. All Rights Reserved.


```

Comment:
It found a weird character and fixed line endings

Reply:
We should exclude these "vendored" files though?